tests/cython/PythonEvalUTest.cxxtest

```diff
@@ -333,8 +281,25 @@ public:
         // Because results are ignored this cannot be tested as of now
         // TS_ASSERT_EQUALS(result, "3");
         TS_ASSERT(true);
+    }
+
+    void testGlobalPythonInitializationFinalization()
+    {
+        logger().debug("[PythonEvalUTest] testGlobalPythonInitializationFinalization()");
+
+        // Initialize and finalize Python.
+        global_python_initialize();
+        global_python_finalize();
 
-        // Cleanup Python.
+        // Initialize and finalize Python.
+	try{
+	    global_python_initialize(); }
+	catch (const std::runtime_error & e){
+	    if (std::string(e.what()) != "reinitialization is not supported by python")
+	        throw e;
+        }
         global_python_finalize();
+
```

Comment:
Test should fail if exception was not thrown

Reply:
I added assert